test/sol/core/TestHeart.sol

```diff
@@ -0,0 +1,133 @@
+pragma solidity ^0.5.0;
+
+import "../../../contracts/modules/core/Heart.sol";
+import "../mocks/MockIRM.sol";
+import "../mocks/MockIERC20.sol";
+import "truffle/Assert.sol";
+
+contract TestHeart {
+
+    Heart public heart;
+    MockIRM public irm;
+    MockIERC20 public ierc20;
+
+    function beforeEach() public {
+        heart = new Heart();
+    }
+
+    constructor() public {
+        irm = new MockIRM();
+        ierc20 = new MockIERC20();
+    }
+
+    function testSetReserveManager() public {
+        address expected = address(irm);
+        heart.setReserveManager(expected);
+
+        Assert.equal(address(heart.getReserveManager()), expected, "heart.setReserveManager() should set reserve manager properly");
+    }
+
+    function testGetReserveManager() public {
+        address expected = address(irm);
+        heart.setReserveManager(expected);
```

Comment:
assert get before set too, otherwise this test case would be exactly the same as testSetReserveManager()

Reply:
maybe we can merge test of getter and setter function into one function ie. testGetSetXxxx()